examples/rvc-app/rvc-common/include/rvc-device.h

```diff
@@ -49,6 +53,12 @@ class RvcDevice
      */
     void Init();
 
+    /**
+     * Sets the device to and idle state, that is either the STOPPED, DOCKED or CHARGING state, depending on physical information.
```

Comment:
```suggestion
     * Sets the device to an idle state, that is either the STOPPED, DOCKED or CHARGING state, depending on physical information.
```

Reply:
I have applied this part of [this commit](https://github.com/project-chip/connectedhomeip/pull/29159/commits/83cd02de4a2c7121fde44e0127f4490f2faf5c79).